content/blog/set-up-apache-skywalking-for-k8s-and-vm-monitoring-in-hpe-greenlake-private-cloud.md

```diff
@@ -0,0 +1,382 @@
+
+In this blog post, I will start choosing one APM tool,  *Apache SkyWalking*, and describe the detailed process how to set it up in HPE GreenLake for Private Cloud Enterprise for monitoring and alerting customer applications. 
+
+## Apache SkyWalking
+
+[Apache SkyWalking](https://skywalking.apache.org/) is an open source APM tool with capabilities for monitoring, tracing and diagnosing distributed system. It’s especially designed for microservices, cloud native and container-based architectures. 
+
+Apache SkyWalking provides a list of agents to be used for building *Java*, *.NET Core*, *PHP*, *NodeJS*, *Golang*, *LUA*, *Rust* and *C++* apps. It provides tracing, metrics analysis, alerting, service mesh observability and visualization.
+
+Apache SkyWalking is lightweight and scalable. It can be easily set up as a *self-managed* APM tool within an on-premises data center. This avoids leasing customer data to third party services and matches well with the strict security restriction in HPE GreenLake for Private Cloud Enterprise environment. 
+
+## Set up Apache SkyWalking for Application Monitoring
+
+I will take the approach to setting up the Apache SkyWalking as a *self-managed* APM tool within the Kubernetes cluster created in HPE GreenLake for Private Cloud Enterprise. This mainly takes into account the security concerns in HPE GreenLake product environment. 
+
+### Prerequisites
+
+Before we start, make sure we meet the following requirements:
+
+* A Kubernetes cluster, being provisioned in HPE GreenLake for Private Cloud Enterprise;
+* The *kubectl* CLI tool, version 1.23 or later, together with the *kubeconfig* files for accessing the Kubernetes cluster; 
+* The *[Helm](https://helm.sh/docs/intro/install/)* CLI tool, version 3.8.1 or later. 
+
+### Deploy Apache SkyWalking
+
+Install Apache SkyWalking using Helm charts with *elasticsearch* as storage:
+
+```markdown
+$ git clone https://github.com/apache/skywalking-kubernetes 
+$ cd skywalking-kubernetes/chart
+$ helm repo add elastic https://helm.elastic.co
+$ helm dep up skywalking
+$﻿ kubectl create ns skywalking
+$ helm install skywalking skywalking –n skywalking \
+--set oap.image.tag=9.2.0 \
+--set oap.storageType=elasticsearch \
+--set ui.image.tag=9.2.0 \
+--set elasticsearch.imageTag=7.17.1 \
+--set elasticsearch.persistence.enabled=true \
+--set elasticsearch.sysctlInitContainer.enabled=false
+```
+
+After running above commands, t﻿he Apache SkyWalking is installed to the Kubernetes cluster's namespace *skywalking*. It creates the *elasticsearch* as the `StatefulSet` object, running a pod on each worker node. It runs the Apache SkyWalking Observability Analysis Platform (OAP) with replicas as 2 to provide high availability. 
```

Comment:
Add "the" prior to "above commands" , take "the" out prior to "Apache Skywalking", and change "to the Kubernetes" to "on the Kubernetes" so it reads:

"After running the above commands, Apache Skywalking is installed on the Kubernetes..."

It's a little unclear as to what "It" refers to in the next two sentences. I believe that in sentence #2, "It" refers to the Apache SkyWalking app that was just installed. But I'm not sure that's the case in the third sentence. I'm thinking that "It" here refers more to the system you have just installed it on. If that's the case, it would be better to spell out the complete subject at the beginning of the 3rd sentence rather than using the pronoun "It".

Reply:
It looks good. Thanks!